che-incubator/che-code
# Remove `che-on-start` extension

Signed-off-by: Artem Zatsarynnyi <azatsary@redhat.com>

### What does this PR do?
Removes the `che-on-start` extension as it becomes redundant after merging https://github.com/che-incubator/che-code/pull/131.
The second part of the extension (opening a terminal on VS Code start) we decided to remove as it doesn't add much value to the user, but makes loading IDE visually slower.

### What issues does this PR fix?
- https://github.com/eclipse/che/issues/21775

### How to test this PR?
1. Start a new DevWorkspace with the following test project https://github.com/azatsarynnyy/java-spring-petclinic/tree/onstart.
2. Check that a new terminal isn't opened automatically after starting VS Code.
